### PR TITLE
docs/events.md: Add new conferences & events since 2020

### PR DESCRIPTION
**Pull request by philipanda**

### CODE REVIEW

File: docs/events.md [hunk 1]
```diff
@@ -4,6 +4,20 @@ Events that TrenchBoot appeared on.
 
 ## Worldwide conferences
 
+* Xen Project Summit 2024
+    * [Slides](https://static.sched.com/hosted_files/xenprojectsummit2024/59/trenchboot_in_xen.pdf)
+* FOSDEM 2024
+    * [Video](https://video.fosdem.org/2024/aw1126/fosdem-2024-3724-trenchboot-project-status-update.av1.webm)
```
Comment (pietrushnic) [inline docs/events.md]: @philipandag, it doesn't seem to be consistent with the rest of the document. Please note other conferences are linked as follows:

```
* <conference_name> <year>
    * Video: [<talk_title>](<talk_video_link>)
    * [Slides](<slides_link>)
```

File: docs/events.md [hunk 2]
```diff
@@ -4,6 +4,20 @@ Events that TrenchBoot appeared on.
 
 ## Worldwide conferences
 
+* Xen Project Summit 2024
+    * [Slides](https://static.sched.com/hosted_files/xenprojectsummit2024/59/trenchboot_in_xen.pdf)
```
Comment (pietrushnic) [inline docs/events.md]: I'm pretty sure there is public video of that.

File: docs/events.md [hunk 3]
```diff
@@ -4,6 +4,20 @@ Events that TrenchBoot appeared on.
...
+* FOSDEM 2024
+    * [Video](https://video.fosdem.org/2024/aw1126/fosdem-2024-3724-trenchboot-project-status-update.av1.webm)
+    * [Slides DRTM status](https://fosdem.org/2024/events/attachments/fosdem-2024-3724-trenchboot-project-status-update/slides/22916/DRTM-presentation-FOSDEM_Crm6zWi.pdf)
+    * [Slides AEM status](https://fosdem.org/2024/events/attachments/fosdem-2024-3724-trenchboot-project-status-update/slides/22916/TrenchBoot_AEM_Project_Status_P8H1V5D.pdf)
```
Comment (pietrushnic) [inline docs/events.md]: a/a

File: docs/events.md [hunk 4]
```diff
@@ -4,6 +4,20 @@ Events that TrenchBoot appeared on.
...
+* Xen Developer & Design Summit 2023
+    * [Slides](https://static.sched.com/hosted_files/xen2023/e6/Paving%20the%20Path%20for%20TrenchBoot%20DRTM%20in%20Xen%20Hypervisor.pdf)
+* Qubes OS Summit 2022
+    * [Video](https://www.youtube.com/watch?v=A9GrlQsQc7Q?start=8152)
+    * [Slides](https://shop.3mdeb.com/wp-content/uploads/2022/09/Qubes-OS-on-modern-Alder-Lake-desktop.pdf)
+* Linux Plumbers Conference 2022
+    * [Slides](https://lpc.events/event/16/contributions/1262/attachments/940/1888/TrenchBoot%20-%20LPC%202022%20-%20Final.pdf)
+    * [Video](https://www.youtube.com/watch?v=FFh3fWvVv0o)
```
Comment (pietrushnic) [inline docs/events.md]: a/a

**Review by pietrushnic — CHANGES_REQUESTED** [review]

File: docs/events.md (hunk 1 above)
Comment (philipanda) [reply docs/events.md]: Thank you

File: docs/events.md [hunk 5]
```diff
@@ -4,6 +4,22 @@ Events that TrenchBoot appeared on.
...
+* FOSDEM 2024
+    * Video: [Trenchboot - project status update](https://video.fosdem.org/2024/aw1126/fosdem-2024-3724-trenchboot-project-status-update.av1.webm)
+    * [Slides (DRTM Status)](slides/DRTM-presentation-FOSDEM_2024.pdf)
+    * [Slides (Trenchboot AEM Status)](slides/TrenchBoot_AEM_Project_Status_fosdem_2024.pdf)
+* Xen Developer & Design Summit 2023
+    * Video: [Paving the Path for TrenchBoot DRTM in Xen Hypervisor](https://www.youtube.com/watch?v=A9GrlQsQc7Q?start=8152)
+    * [Slides](slides/Paving_the_Path_for_TrenchBoot_DRTM_in_Xen_Hypervisor.pdf)
+* Qubes OS Summit 2022
+    * Video: [TrenchBoot - The only AEM-way to boot Qubes OS](https://www.youtube.com/watch?v=A9GrlQsQc7Q&t=17440s)
+    * [Slides]()
```
Comment (philipanda) [inline docs/events.md]: I could not find the slides for this presentation.

File: docs/events.md (hunk 1 above)
Comment (philipanda) [reply docs/events.md]: https://github.com/TrenchBoot/TrenchBoot.github.io/compare/b81d13e816bf7fd747c61bb0eea65bbdc80c102d..e77edbe0711d15428aad1ef9d300428d97b86887#diff-e070a13a5415feaf83554427f172fb145dcb7c57e396319bcc967c496a3a43f8

File: docs/events.md (hunk 5 above)
Comment (pietrushnic) [reply docs/events.md]: @miczyg1 can you upload slides here: https://cfp.3mdeb.com/qubes-os-summit-2022/talk/3PTQJU/

Comment (pietrushnic) [reply docs/events.md]: @philipandag we can do two things, wait for @miczyg1 or remove this conference, to second PR, so we will not forget to resolve the issue. Then I would be able to approve whole PR and merge.

Comment (philipanda) [reply docs/events.md]: I have found the slides, turns out they were available at [3mdeb.com](https://3mdeb.com/events) all along. I have missed them and assumed I need to find them in the web - which I did not succeed in.
https://github.com/TrenchBoot/TrenchBoot.github.io/compare/10b0c6f..ccbdcd7

File: docs/events.md (hunk 2 above)
Comment (philipandag) [reply docs/events.md]: https://github.com/TrenchBoot/TrenchBoot.github.io/compare/b81d13e816bf7fd747c61bb0eea65bbdc80c102d..e77edbe0711d15428aad1ef9d300428d97b86887#diff-e070a13a5415feaf83554427f172fb145dcb7c57e396319bcc967c496a3a43f8R10-R11

Comment (philipanda) [reply docs/events.md]: https://github.com/TrenchBoot/TrenchBoot.github.io/compare/b81d13e816bf7fd747c61bb0eea65bbdc80c102d..e77edbe0711d15428aad1ef9d300428d97b86887#diff-e070a13a5415feaf83554427f172fb145dcb7c57e396319bcc967c496a3a43f8R7-R8

File: docs/events.md (hunk 3 above)
Comment (philipanda) [reply docs/events.md]: There are two parts of this presentation - and two sets of slides. Would this deviation from the rest of the document be acceptable?

https://github.com/TrenchBoot/TrenchBoot.github.io/compare/b81d13e816bf7fd747c61bb0eea65bbdc80c102d..e77edbe0711d15428aad1ef9d300428d97b86887#diff-e070a13a5415feaf83554427f172fb145dcb7c57e396319bcc967c496a3a43f8R10-R16

File: docs/events.md (hunk 4 above)
Comment (philipanda) [reply docs/events.md]: https://github.com/TrenchBoot/TrenchBoot.github.io/compare/b81d13e816bf7fd747c61bb0eea65bbdc80c102d..e77edbe0711d15428aad1ef9d300428d97b86887#diff-e070a13a5415feaf83554427f172fb145dcb7c57e396319bcc967c496a3a43f8R20-R22

File: docs/events.md [hunk 6]
```diff
@@ -20,6 +20,8 @@ Events that TrenchBoot appeared on.
 * Linux Plumbers Conference 2022
     * Video: [TrenchBoot Update](https://www.youtube.com/watch?v=FFh3fWvVv0o)
     * [Slides](slides/TrenchBoot%20-%20LPC%202022%20-%20Final.pdf)
+* FOSDEM 2021
+    * Video: [Secure Upgrades with DRTM](https://video.fosdem.org/2021/D.firmware/firmware_suwd.webm)
```
Comment (philipanda) [inline docs/events.md]: I could not find the slides for this presentation

Comment (philipanda) [reply docs/events.md]: I think it would be best to do as @pietrushnic said and move it to a seperate PR

Comment (philipanda) [reply docs/events.md]: https://github.com/TrenchBoot/TrenchBoot.github.io/pull/34

File: docs/events.md (hunk 3 above)
Comment (pietrushnic) [reply docs/events.md]: Yes

**Review by pietrushnic — APPROVED** [review]

File: docs/events.md (hunk 5 above)
Comment (miczyg1) [reply docs/events.md]: Uploaded them as a resource to both of the talks:
https://cfp.3mdeb.com/qubes-os-summit-2022/talk/3PTQJU/
https://cfp.3mdeb.com/qubes-os-summit-2022/talk/QXZBDZ/

**Pull request merged**